.github/workflows/release-native.yml

```diff
@@ -83,12 +80,59 @@ jobs:
         with:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
       - name: Build and push
+        id: build
         uses: docker/build-push-action@v6
         with:
           context: .
+          platforms: ${{ matrix.platform }}
           push: true
-          tags: ${{ steps.meta.outputs.tags }}
+          outputs: type=image,"name=scalameta/scalafmt",push-by-digest=true,name-canonical=true
```

Comment:
why do we not need tags here?

Reply:
This is because docker build is split into two steps: first is building and pushing by digest (each digest is only for one platform), and then it will combine the digests and finally create the multi-arch image.